checkstyle.xml

```diff
@@ -5,7 +5,6 @@
 
 <module name="Checker">
   <module name="SuppressWarningsFilter"/>
-  <module name="NewlineAtEndOfFile"/>

```

Comment:
?

Reply:
Now the diff here is extra confusing, because you've re-added this in a different place. When preparing to make a pull request, you should look over the diff and try to remove anything that looks out of place.